### ISSUE TITLE
$element.data triggers incorrect usage warning

### ISSUE THREAD

**Issue by pedromorais** [padolsey-archive/jquery.lint--old#9]
Using the following code:

```
    $('#testdiv').data('value', 5);
```

Triggers the following warning:

<pre>
data(...) called incorrectly
jquery.lint.js (linha 32)
More info:
Collection: [div#testdiv]
Location:
@file:///Users/pedromorais/Desktop/test.html:12
jquery.lint.js (linha 41)
You passed: ["value", 5]
Available signatures include:
data(key, value)
data(obj)
data(key)
data()
</pre>

Using latest version available on github.

**Comment by padolsey** [comment]
Thanks for raising this issue.

Fixed in the latest version: http://github.com/jamespadolsey/jQuery-Lint/commit/2947794e7e2b10e2254cf5a7389c4bc04c4257a3
